src/Common/DateLUTImpl.h

```diff
@@ -71,14 +71,14 @@ class DateLUTImpl
     // Same as above but select different function overloads for zero saturation.
     STRONG_TYPEDEF(UInt32, LUTIndexWithSaturation)
 
-    static inline LUTIndex normalizeLUTIndex(UInt32 index)
```

Comment:
What is the motivation of removing `inline` from this file?

Reply:
Functions defined inside class definitions are implicitly inline ([SO](https://stackoverflow.com/questions/9192077/is-inline-implicit-in-c-member-functions-defined-in-class-definition)). I find `inline` keywords slightly distracting in such contexts, so I decided to remove it.